code/datums/brain_damage/severe.dm

```diff
@@ -69,9 +69,6 @@
 		paralysis_type = pick("full","left","right","arms","legs","r_arm","l_arm","r_leg","l_leg")
```

Comment:
```suggestion
		paralysis_type = pick("left","right","arms","legs","r_arm","l_arm","r_leg","l_leg")
```

Reply:
don't remove the "full" location entirely, just remove it from the list of randomly selectable locations (and add a comment saying that you've done this). admins and such might want to be able to give people full paralysis brain traumas.

Reply:
Yikes. I should have caught that.